tensorflow_cloud/validate.py

```diff
@@ -150,6 +153,53 @@ def _validate_cluster_config(chief_config, worker_count, worker_config):
             "Received {}.".format(worker_count)
         )
 
+def _validate_labels(labels):
+    """Validates labels conform guidelines at 
+       https://cloud.google.com/ai-platform/training/docs/resource-labels"""
+    if not labels:
+        print("No label for the job. Consider creating labels"
```

Comment:
Please reword this as "No labels provided for the training job. Please consider creating labels to help with retrieval of job information. Please see https://cloud.google.com/ai-platform/training/docs/resource-labels for more information."

+ All of this should live in gcp.py and invoked from here.